src/tools/bin/plot_chargeConservation.py

```diff
@@ -0,0 +1,209 @@
+#!/usr/bin/env python
+#
+# Copyright 2015 Richard Pausch
+#
+# This file is part of PIConGPU.
+#
+# PIConGPU is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# PIConGPU is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with PIConGPU.
+# If not, see <http://www.gnu.org/licenses/>.
+#
+
+import argparse
+import numpy as np
+import h5py
+import matplotlib.pyplot as plt
+
+
+__doc__ = '''
+This program reads electric field and charge density data
+from hdf5 files created by PIConGPU and checks charge conservation.
+
+Three slice plots show the error in $div(E) - rho/epsilon_0$
+normalized to the maximum charge of all species simulated.
```

Comment:
maybe we should specify `maximum [per-species] charge in the simulation`. else it could sound like the sum